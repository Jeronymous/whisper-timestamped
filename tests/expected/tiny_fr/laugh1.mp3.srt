1
00:00:00,060 --> 00:00:01,760
Je ne suis pas en train de faire Ã§a.

