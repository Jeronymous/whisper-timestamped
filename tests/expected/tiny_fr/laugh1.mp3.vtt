WEBVTT

00:00.060 --> 00:01.760
Je ne suis pas en train de faire Ã§a.

