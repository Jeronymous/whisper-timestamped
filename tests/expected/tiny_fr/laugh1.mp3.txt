Je ne suis pas en train de faire Ã§a.
